neondatabase/website
# chore: add Encore logo

This pull request brings Encore partner logo.

Steps to check:
- open preview [partners](https://neon-next-git-encore-logo-neondatabase.vercel.app/partners), [pricing](https://neon-next-git-encore-logo-neondatabase.vercel.app/pricing) and [enterprise](https://neon-next-git-encore-logo-neondatabase.vercel.app/enterprise) pages
- make sure that everything looks and feels as expected
